pkg/router/types.go

```diff
@@ -180,6 +184,13 @@ func (f *handlerFactories) exists(name string) bool {
 	return ok
 }
 
+func GetDefaultRouteHandler() string {
+	if GetMakeHandlerFunc(defaultRouteHandlerName) == nil {
```

Comment:
maybe no need to get handler here, just check `defaultRouteHanlderName` is not empty string is enough

Reply:
cc @nejisama

Reply:
or, just set `defaultRouteHandlerName = types.DefaultRouteHandler` at line 51?

Reply:
If static initialized, what would happen if it's unset to "" ? (As above comment)

Reply:
Sorry, @fibbery , It's my mistake, there is no condition that user could reset handler name to empty string since it's not exported. 

So there is no need to use this method, static initializing `defaultRouteHandlerName = types.DefaultRouteHandler` is LGTM